src/utils/Wallet.ts

```diff
@@ -0,0 +1,141 @@
+import { generateMnemonic, mnemonicToSeed } from 'bip39';
+import { providers, utils, Wallet as EthersWallet } from 'ethers';
+import { hdkey } from 'ethereumjs-wallet';
+import { privateToAddress } from 'ethereumjs-util';
+import { ServiceResponse } from "../services/core/service-response";
+import { NetworkUtil } from './networks';
+
+
+
+export type Account = {
+  address: string
+  privateKey: string
+}
+
+export type WalletSecret = {
+  address: string
+  mnemonic: string
+  wallet: EthersWallet
+}
+
+export type WalletInfo = {
+  address: string
+  balance: string
+  latestTransactions: any
+}
```

Comment:
Can we add these types in a different folder so we can use them elsewhere also

Reply:
We are maintaining types and enums on respective utils in frontened.